main.tf

```diff
@@ -329,3 +329,17 @@ resource "aws_vpc_peering_connection" "this" {
 
   tags = "${merge(var.tags, var.vpc_tags, map("Name", format("%s-to-%s", var.name, element(var.vpc_peering_with_name, count.index))))}"
 }
+
+################################
+# Bucket S3 ## We use it for k8s
```

Comment:
maybe instead of commenting this as it's specifically for k8s we should mention that it's used for metadata on implementations.